docs/developers/nft-tokens.md

```diff
@@ -577,7 +634,7 @@ Assign `ESDTRoleNFTCreate` and `ESDTRoleNFTAddQuantity` roles to an address. You
 
 **Step 4: Create NFT**
 
-Fill all the attributes as you think.
+To see more information regarding the fields used in this example you can go [here](/developers/esdt-tokens#example).
```

Comment:
_Alternate option to this phrase, if you like:_
Now, the NFT creation transaction for the example case defined [here] looks like this: